src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.css

```diff
@@ -168,3 +152,42 @@ a {
         color: #13709e;
         text-decoration: underline;
     }
+
+.showRawException {
+    font-size: 1.2em;
+    cursor: pointer;
```

Comment:
Any reason not to keep that one as a link? Links are also automatically tab stops. Might need less CSS that way?